package.json

```diff
@@ -147,6 +147,7 @@
     "test-expressions": "build/run-node test/expression.test.js",
     "test-flow": "build/run-node build/generate-flow-typed-style-spec && flow .",
     "test-cov": "nyc --require=@mapbox/flow-remove-types/register --reporter=text-summary --reporter=lcov --cache run-s test-unit test-expressions test-query test-render",
+    "test-style-spec": "cd src/style-spec && npm run build && npm test",
```

Comment:
I think `npm run build` should be a part of the test there, e.g. in a `pretest` script.